### DISCUSSION TITLE
Upgrading GitHub Enterprise from version 2.16 to 3.8

### DISCUSSION THREAD

**Discussion by Waqar01** [community/community#137901] [Enterprise]
### Select Topic Area

Question

### Body

Dear community,

We are currently using GitHub Enterprise version 2.16 and are planning to migrate to GitHub Enterprise Cloud. According to the documentation, we need to upgrade our GitHub Enterprise server to version 3.8 before we can proceed with the migration.

We would appreciate your guidance and assistance with the in-place upgrade process from version 2.16 to 3.8, as well as the subsequent migration to GitHub Enterprise Cloud.

Any feedback or guidance you can provide would be greatly appreciated.

Thank you,

**Comment by queenofcorgis** [comment]
@Waqar01 have you checked our our Upgrade Server Planner form: https://support.github.com/enterprise/server-upgrade? 

This should help you plan you upgrade, but we also recommend reaching out on our [Support page](https://support.github.com/contact).
